langlib/lang.annotations/src/main/ballerina/src/lang.annotations/annotations.bal

```diff
@@ -68,3 +68,5 @@ public type StrandData record {|
 # Denotes new Strand execution semantics.
 public const annotation StrandData strand on source worker;
 
+# This annotation specify icon meta-data about types and functions.
```

Comment:
For consistency should we say something like

```suggestion
# Denotes icon metadata related to types and functions.
```

Reply:
Fixed